### ISSUE TITLE
Autoencoder with Replicated Softmax Layer

### ISSUE THREAD

**Issue by parthg** [nitishsrivastava/deepnet#55]
Hi
I want to train autoencoder with replicated softmax layer. 

So far I can think of training RBM with replicated softmax and then while constructing autoencoder:
- convert the output layer to be softmax, 
- make a copy of the input data normalized with |D| (i.e. length of documents) for data_field in output later
- backpropagation with GD

I have following questions:
- Is this the correct/best way with Deepnet?
- Is there anyway, I can use the normalised version of the input data without making a copy and using the "tied_to" feature.

Many thanks in advance. Any pointer will be very useful. Thanks.
